android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TableViewHolder.java

```diff
@@ -253,16 +253,8 @@ public void bind(final TableViewRowProxy proxy, final boolean selected)
 		// Set font and text color for title.
 		TiUIHelper.styleText(this.title, properties.getKrollDict(TiC.PROPERTY_FONT));
 
-		// Determine title color based on background.
-		final int tableBackgroundColor =
-			TiConvert.toColor(tableViewProxy.getProperties(), TiC.PROPERTY_BACKGROUND_COLOR);
-		final int rowBackgroundColor =
-			TiConvert.toColor(properties, TiC.PROPERTY_BACKGROUND_COLOR);
-		final int backgroundColor = rowBackgroundColor != Color.TRANSPARENT ? rowBackgroundColor : tableBackgroundColor;
-		final int defaultTitleColor = backgroundColor == Color.TRANSPARENT || backgroundColor < (Color.BLACK / 2)
-			? Color.WHITE : Color.BLACK;
-		final int titleColor = properties.containsKey(TiC.PROPERTY_COLOR)
-			? TiConvert.toColor(properties.getString(TiC.PROPERTY_COLOR)) : defaultTitleColor;
+		// Set title color.
+		final int titleColor = TiConvert.toColor(properties.optString(TiC.PROPERTY_COLOR, "black"));
```

Comment:
Instead of defaulting it to "black" (which is hard to read using the default dark theme), we should let it use the theme's default primary text color. Can we get away with not setting the text color at all like this?
```java
if (properties.containsKeyAndNotNull(TiC.PROPERTY_COLOR)) {
	final int titleColor = TiConvert.toColor(properties.optString(TiC.PROPERTY_COLOR, "black"));
	this.title.setTextColor(titleColor);
}
```